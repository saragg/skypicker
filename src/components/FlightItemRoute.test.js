import React from 'react';
import Route from './FlightItemRoute';
import renderer from 'react-test-renderer';
import MuiThemeProvider from 'material-ui/styles/MuiThemeProvider';

test('Renders correctly', () => {
  const route = {
    cityFrom: "Barcelona",
    flyFrom: "Tokio",
    countryFrom: {
      code: 'ES',
    },
    countryTo: {
      code: 'JP',
    },
    fly_duration: "18h 30m",
    return_duration: "20h 45m",
    nightsInDest: 3,
    price: 170,
  }

  const tree = renderer.create(
    <MuiThemeProvider>
      <Route route={route}/>
    </MuiThemeProvider>
  ).toJSON();

  expect(tree).toMatchSnapshot();
});