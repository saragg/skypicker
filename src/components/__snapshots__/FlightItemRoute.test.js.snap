// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders correctly 1`] = `
<div
  className="route"
>
  <div>
    Barcelona
    (
    Tokio
    )
    <div
      className="time"
    >
      Invalid date
    </div>
  </div>
  <div>
    <span
      className="fa fa-long-arrow-right"
      onMouseEnter={[Function]}
      onMouseLeave={[Function]}
      style={
        Object {
          "color": "#666",
          "display": "inline-block",
          "fontSize": 24,
          "muiPrepared": true,
          "position": "relative",
          "transition": "all 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms",
          "userSelect": "none",
        }
      }
    />
  </div>
  <div>
    (
    )
    <div
      className="time"
    >
      Invalid date
    </div>
  </div>
</div>
`;
